### PR TITLE
added wishlist page

### PR DESCRIPTION
**Pull request by saicharan-gouru**
wishlist

### CODE REVIEW

**Comment by govardhanreddyc79** [comment]
looks good to me.

File: Pages/Wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+    margin: 0px;
+}
+
+:root {
+    --primary-color: #2563EB;
+    --gold: rgb(253, 196, 52);
+    --white: #ffff;
+    --black: rgba(0, 0, 0, 0.8);
```
Comment (aniketsingh13) [inline Pages/Wishlist/wishlist.css:12]: use hex code for consistency

File: Pages/Wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.hero-header {
+    display: flex;
+    align-items: center;
```
Comment (aniketsingh13) [inline Pages/Wishlist/wishlist.css:17]: utility can be used if these things repeat on your code

File: Pages/Wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        <p class="hero-category-name">Grocery</p>
+        <input class="input-textbox" type="text" placeholder="Search here..." name="text" />
+        <button class="button primary-yellow">Login</button>
+        <a><span class="fab fa-solid fa-cart-shopping"></span></a>
+        <a><span class="fab fa-solid fa-heart active"></span></a>
+    </header>
+
+    <main>
+        <h1>
+
```
Comment (aniketsingh13) [inline Pages/Wishlist/wishlist.html]: remove spaces

File: Pages/Wishlist/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                    </div>
+                </div>
+                <div class="container2 ">
+                    <button class="btn ">Add to cart</button>
+                    <i class="far fa-heart "></i>
+                    <i class="fas fa-share "></i>
+                    <i class="fas fa-ellipsis-v "></i>
+                </div>
+            </div>
+
```
Comment (aniketsingh13) [inline Pages/Wishlist/wishlist.html:48]: here also

File: Pages/Wishlist/wishlist.html [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
```
Comment (aniketsingh13) [inline Pages/Wishlist/wishlist.html:64]: here also

**Review by aniketsingh13 — APPROVED** [review]
please consider the changes and remove the unnecessary space in file . other thing look good to me

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

**Pull request merged**